nixos/modules/services/networking/murmur.nix

```diff
@@ -343,6 +351,7 @@ in
         ProtectKernelModules = true;
         ProtectKernelTunables = true;
         ProtectSystem = "full";
+        ReadWritePaths = [ cfg.stateDir ];
```

Comment:
This won't have any effect at the moment, and also goes beyond what the patch says. Please remove.